docs/Config.md

```diff
@@ -376,6 +374,28 @@ Alternatively you may have bold fonts disabled in your terminal, in which case e
 
 If you're still having trouble please raise an issue.
 
+## Custom Author Color
+
+Lazygit will assgin a random color for every commit author in the commits pane by default.
+
+You can customize the color in case you're not happy with the randomly assigned one:
+
+```yaml
+gui:
+  authorColors:
+    'John Smith': '#ff0000 # use red for John Smith
+```
+
+You can use wildcard to set a unified color in case your are lazy to customize the color for every author or you are just want a single color for all/other authors:
```

Comment:
```suggestion
You can use wildcard to set a unified color in case your are lazy to customize the color for every author or you just want a single color for all/other authors:
```